### PR TITLE
python310Packages.tempest: add missing input

### PR DESCRIPTION
**Pull request by fabaff**
###### Description of changes
Fix build (https://hydra.nixos.org/build/208857095)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 215993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.tempest</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python310Packages.tempest</li>
  </ul>
</details>

**Review by SuperSandro2000 — COMMENTED** [review]
Because of the formatting I can't see what inputs exactly changed.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 215993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.tempest</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python310Packages.tempest</li>
  </ul>
</details>

File: pkgs/development/python-modules/tempest/default.nix [hunk 1]
```diff
@@ -36,9 +36,13 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-aEtBAE3p+HVw/macwZtKo20mSJctrsIN7idqWe6Dvtc=";
+    hash = "sha256-aEtBAE3p+HVw/macwZtKo20mSJctrsIN7idqWe6Dvtc=";
   };
 
+  nativeBuildInputs = [
+    pbr
```
Comment (piegamesde) [inline pkgs/development/python-modules/tempest/default.nix:44]: It's still listed in `propagatedBuildInputs`, is that intended (from the commit message I'd guess no)

**Comment by piegamesde** [comment]
> Because of the formatting I can't see what inputs exactly changed.

The noisy changes are all cleanly separated out into a single commit, which is easy to ignore when reviewing commit by commit (Ref https://github.com/NixOS/nixpkgs/pull/216973/commits/ba948164d35d75e3ff107d90ca9e6cea77194118#r1111337733).

File: pkgs/development/python-modules/tempest/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/tempest/default.nix:44]: No, should be removed.

**Comment by SuperSandro2000** [comment]
I am pretty sure it was one commit before and they got splitted into multiple in the force push after the comment.

ref https://github.com/NixOS/nixpkgs/commit/0b5ee4b5b0287d97e0f661ebbb9c03c94bc8ab18

File: pkgs/development/python-modules/tempest/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/tempest/default.nix:44]: this is used at runtime https://github.com/openstack/tempest/blob/1569290be06e61d63061ae35a997aff0ebad68f1/tempest/version.py#L16-L18

**Comment by piegamesde** [comment]
@SuperSandro2000 oh, I see. Apologies for the snark then.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 215993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.tempest</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python310Packages.tempest</li>
  </ul>
</details>

**Pull request closed without merging**